Android drawables in nuget packages created from Uno libraries cause Windows Target build failure

### Current behavior 🐛

Build with only a net10.0-windows10.0.26100 target framework fails with the following errors:

Note these are all Android drawable files and this is a Windows build

Build started at 17:52...
1>------ Build started: Project: TopLevelControls, Configuration: Debug Any CPU ------
1>  TopLevelControls -> D:\GitHub.Public\UnoUINugetInAppBuildIssue\TopLevelControls\bin\Debug\net10.0-ios\TopLevelControls.dll
1>  TopLevelControls -> D:\GitHub.Public\UnoUINugetInAppBuildIssue\TopLevelControls\bin\Debug\net10.0-android\TopLevelControls.dll
1>  TopLevelControls -> D:\GitHub.Public\UnoUINugetInAppBuildIssue\TopLevelControls\bin\Debug\net10.0-browserwasm\TopLevelControls.dll
1>C:\Program Files\dotnet\sdk-manifests\10.0.100\microsoft.net.workload.mono.toolchain.current\10.0.103\WorkloadManifest.targets(124,5): warning : @(NativeFileReference) is not empty, but the native references won't be linked in, because neither $(WasmBuildNative), nor $(RunAOTCompilation) are 'true'. NativeFileReference=C:\Users\vsoft\.nuget\packages\skiasharp.nativeassets.webassembly\3.119.1\buildTransitive\netstandard1.0\libSkiaSharp.a\3.1.56\st\libSkiaSharp.a
1>  TopLevelControls -> D:\GitHub.Public\UnoUINugetInAppBuildIssue\TopLevelControls\bin\Debug\net10.0-windows10.0.26100\TopLevelControls.dll
2>------ Build started: Project: UnoUINugetInAppBuildIssue, Configuration: Debug Any CPU ------
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(30,12,30,24): warning CS8618: Non-nullable field 'dispatcher' must contain a non-null value when exiting constructor. Consider adding the 'required' modifier or declaring the field as nullable.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(39,27,39,57): warning CS8601: Possible null reference assignment.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Initialisation\Bootstrapper.cs(29,40,29,44): warning CS8600: Converting null literal or possible null value to non-nullable type.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(85,34,85,51): warning CS8622: Nullability of reference types in type of parameter 'state' of 'lambda expression' doesn't match the target delegate 'SendOrPostCallback' (possibly because of nullability attributes).
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(109,42,109,46): warning CS8625: Cannot convert null literal to non-nullable reference type.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(106,21,106,38): warning CS8622: Nullability of reference types in type of parameter 'state' of 'lambda expression' doesn't match the target delegate 'SendOrPostCallback' (possibly because of nullability attributes).
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(30,12,30,24): warning CS8618: Non-nullable field 'dispatcher' must contain a non-null value when exiting constructor. Consider adding the 'required' modifier or declaring the field as nullable.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(39,27,39,57): warning CS8601: Possible null reference assignment.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Initialisation\Bootstrapper.cs(29,40,29,44): warning CS8600: Converting null literal or possible null value to non-nullable type.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(85,34,85,51): warning CS8622: Nullability of reference types in type of parameter 'state' of 'lambda expression' doesn't match the target delegate 'SendOrPostCallback' (possibly because of nullability attributes).
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(109,42,109,46): warning CS8625: Cannot convert null literal to non-nullable reference type.
2>D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\Services\UIDispatcher.cs(106,21,106,38): warning CS8622: Nullability of reference types in type of parameter 'state' of 'lambda expression' doesn't match the target delegate 'SendOrPostCallback' (possibly because of nullability attributes).

Thousands of lines like this 
Full error list is attached

windows10.0.26100\Vssl.Framework.SampleLibary\obj\Debug\net10.0-android\lp\95\jl\res\drawable-xhdpi-v4\abc_list_focused_holo.9.png". Could not find a part of the path 'D:\GitHub.Public\UnoUINugetInAppBuildIssue\UnoUINugetInAppBuildIssue\bin\Debug\net10.0-windows10.0.26100\TopLevelControls\bin\Debug\net10.0-windows10.0.26100\Vssl.Framework.SampleLibary\obj\Debug\net10.0-android\lp\95\jl\res\drawable-xhdpi-v4\abc_list_focused_holo.9.png'.
========== Build: 1 succeeded, 1 failed, 3 up-to-date, 0 skipped ==========
========== Build completed at 18:01 and took 09:23.321 minutes ==========

Likely related to these file in the nuget package

<img width="1173" height="1028" alt="Image" src="https://github.com/user-attachments/assets/a348f098-de2d-417b-a895-b0bdfaaf4165" />


### Expected behavior 🎯

Should build to success

### How to reproduce it (as minimally and precisely as possible) 🔬

Instructions to reproduce
The code is in the git hub repository:
https://github.com/BrianDT/UnoUINugetInAppBuildIssue

The sample has a complex project structure designed to mimic the structured layout of much larger projects.
The steps below recreate the upgrade steps that caused the issue.

The project SampleFrameworkInterfaces, SampleConverters and SampleLibrary create nuget packages. You will likely need to edit the last lines in the project files to adjust the nuget command location and the destination of the nuget packages.

Unload the TopLevelControls and UnoUINugetInAppBuildIssue projects
Edit the global.json file and switch to Uno.Sdk: 6.4.58
In the projects SampleFrameworkInterfaces, SampleConverters and SampleLibrary 
	Uncomment the net9 target frameworks and comment the net 10 target frameworks
	Uncomment the 9.x version and comment the 10.x version
In the Directory.Packages.props uncomment the net 9 versions and comment the net 10 versions
Release build the SampleFrameworkInterfaces, SampleConverters and SampleLibrary in this sequence.
Re-edit the global.json file and switch back to Uno.Sdk: 6.5.31
Reverse the steps above
Release build the net 10 versions on Uno.Sdk: 6.5.31 of SampleFrameworkInterfaces, SampleConverters and SampleLibrary in this sequence.

Switch to Debug
Reload the TopLevelControls and UnoUINugetInAppBuildIssue projects
It may be necessary to do a manual dotnet restore in these project folders

### Workaround 🛠️

_No response_

### Renderer 🎨

- [x] Skia
- [ ] Native

### Affected platforms 📱💻🖥️

Windows App SDK, All platforms 🌍

### Uno.Sdk version (and other relevant versions) 📦

Uno SDK 6.4.58 and 6.5.31

### IDE version 🧑‍💻

Visual Studio Insiders [11506.43]

### Anything else we need to know? 💬

Related to https://github.com/unoplatform/uno/discussions/22640

Full error list

[Error List.txt](https://github.com/user-attachments/files/25270140/Error.List.txt)

--- a/src/Uno.Sdk/targets/Uno.DefaultItems.targets
+++ b/src/Uno.Sdk/targets/Uno.DefaultItems.targets
@@ -6,7 +6,7 @@
 		<DefaultItemExcludes Condition="!$(DefaultItemExcludes.Contains('obj/**'))">$(DefaultItemExcludes);obj/**</DefaultItemExcludes>
 
 		<!-- Exclude the Stories folder when in Optimize (aka Release) build configuration -->
-		<DefaultItemExcludes Condition=" '$(Optimize)' == 'true' AND '$(HotDesignStoriesFolder)' != '' ">$(HotDesignStoriesFolder)/**</DefaultItemExcludes>
+		<DefaultItemExcludes Condition=" '$(Optimize)' == 'true' AND '$(HotDesignStoriesFolder)' != '' ">$(DefaultItemExcludes);$(HotDesignStoriesFolder)/**</DefaultItemExcludes>
 	</PropertyGroup>
 
 	<!-- Default Includes-->

--- a/src/Uno.Sdk/targets/Uno.Sdk.After.targets
+++ b/src/Uno.Sdk/targets/Uno.Sdk.After.targets
@@ -5,7 +5,7 @@
 	</ItemGroup>
 
 	<!-- Sanity check ensure that platform files for other targets are not included -->
-	<ItemGroup Condition=" $(_IsUnoSingleProjectAndLegacy) == 'true' and $(PlatformsProjectFolder) != '' and Exists($(PlatformsProjectFolder))">
+	<ItemGroup Condition=" '$(TargetFramework)' != '' ">
 		<Compile Remove="@(_IgnorePlatformFiles)" />
 		<Page Remove="@(_IgnorePlatformFiles)" />
 		<Content Remove="@(_IgnorePlatformFiles)" />

